src/areas/game/game_display.tsx

```diff
@@ -77,6 +91,8 @@ export class GameDisplay extends React.Component<GameDisplayProps, GameDisplaySt
             return (
                 <>
                     <button style={{position: "absolute"}} onClick={this.testOnClick}>TEST BUTTON</button>
+                    <button style={{position:"absolute",left:"69%", }} onClick={this.toggleMusic}><img id={"speakerImage"} src={"images/audio/audioOff.png"}/></button>
```

Comment:
move this into a css file, create one for game_display.tsx

also `<img id={..` does not need curly braces, same with below